### PR TITLE
make standalone runnable

### PR DESCRIPTION
**Pull request by domdorn**
I refactored the code a bit to make this a standalone runnable. 

now takes a parameter to turn the switch on or off (true/false) and stores the settings in a properties-file. 
Thus its possible to simply copy the jar a few times to control multiple switches via cron/scripts/whatever. 

Long-term goal would be to create a graal native-image out of it, but haven't looked into that yet.

### CODE REVIEW

File: src/main/java/com/github/k4czp3r/tapop100/Main.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            new BufferedReader(new InputStreamReader(System.in));
+        Optional<String> maybeIP = Optional.ofNullable(prop.getProperty("ip")).filter(s -> !s.equals("null"));
+        Optional<String> maybeEmail = Optional.ofNullable(prop.getProperty("email")).filter(s -> !s.equals("null"));
+        Optional<String> maybePassword = Optional.ofNullable(prop.getProperty("password")).filter(s -> !s.equals("null"));
+
+        if(maybeIP.isEmpty()) {
+            System.out.println("No IP of PowerPlug specified. Please adjust tapop100.properties");
+        }
+        if(maybeEmail.isEmpty()) {
+            System.out.printf("No TP-Email-Address specified. Please adjust tapop100.properties");
```
Comment (JohnAZoidberg) [inline src/main/java/com/github/k4czp3r/tapop100/Main.java]:
```suggestion
            System.out.println("No TP-Email-Address specified. Please adjust tapop100.properties");
```

**Review by JohnAZoidberg — APPROVED** [review]
Wow, this works! Thanks so much @K4CZP3R and @domdorn!

**Comment by domdorn** [comment]
Great to hear that @JohnAZoidberg ! @K4CZP3R can we merge this?

**Comment by kacper-serewis** [comment]
Yes! Sorry for the delay, when I get home I will look at it.

**Review by kacper-serewis — APPROVED** [review]

**Comment by kacper-serewis** [comment]
Merged!

**Comment by jahtz** [comment]
Amazing work! Would it be possible to add a "read state" argument for the standalone cli? Would be amazing to see if some plugs in another room are on or off.

**Pull request merged**